mantle/cosa/cosa_v1.go

```diff
@@ -44,6 +44,7 @@ type Build struct {
 	GitDirty                  string                `json:"coreos-assembler.config-dirty,omitempty"`
 	ImageInputChecksum        string                `json:"coreos-assembler.image-input-checksum,omitempty"`
 	InputHasOfTheRpmOstree    string                `json:"rpm-ostree-inputhash"`
+	Meta                      interface{}           `json:"meta,omitempty"`
```

Comment:
Should this be of type `Meta`?

Reply:
FWIW, I think no. That's coming from the schema definition and its generated code.

Reply:
Here is the result of validating the jsonschema generated in `cosa_v1.go`:

```
jsonschema.exceptions.SchemaError: 'meta' is not valid under any of the given schemas

Failed validating 'anyOf' in metaschema['properties']['properties']['additionalProperties']['properties']['type']:
    {'anyOf': [{'$ref': '#/definitions/simpleTypes'},
               {'items': {'$ref': '#/definitions/simpleTypes'},
                'minItems': 1,
                'type': 'array',
                'uniqueItems': True}]}

On schema['properties']['meta']['type']:
    'meta'
```

Removing the meta type no longer causes the error for validation:

```
    "meta": {
      "$id":"#/properties/meta",
      "type":"meta",
      "title":"Meta"
     },
```

and the tests+govet+gofmt succeed:

```
<snip/>
Checking gofmt...
Checking govet...
Running commands...
 Running kola...
 Running kolet...
 Running ore...
 Running plume...
Success
$
```

Reply:
Interestingly enough the [additional definition looks valid to me](https://github.com/coreos/coreos-assembler/pull/1663/files#diff-62fcc7d8622e3cb439eabdd9dc1c60bcR156-R166).

Reply:
Running a new `./build` actually generates a different definition and seems to work:

```
+       Meta                      map[string]interface{} `json:"meta,omitempty"`
```

Reply:
@ashcrow here's a fixup that highlights the problem:
https://github.com/darkmuggle/coreos-assembler/commit/27b55be33a5b548e3d3a44b33a6ad2fb75aa655e